### PR TITLE
Update `Operator.dependencies` to be a property

### PR DESCRIPTION
**Pull request by karlhigley**
Companion PR to NVIDIA-Merlin/core#33 to account for the changes there

### CODE REVIEW

**Comment by karlhigley** [comment]
Tests are failing here because CI is using the latest released version of `merlin-core`, not the one from the corresponding PR

**Review by jperez999 — CHANGES_REQUESTED** [review]

File: examples/advanced-ops-outbrain/02-ETL-with-NVTabular.ipynb [hunk 1]
```diff
@@ -3,6 +3,8 @@
   {
```
[the remark this answers is not in the document]
Comment (karlhigley) [reply examples/advanced-ops-outbrain/02-ETL-with-NVTabular.ipynb:3]: Just the property change, will update

**Review by benfred — APPROVED** [review]
lgtm - though we probably should install the latest merlin-core too

**Comment by karlhigley** [comment]
Superseded by #1441

**Pull request closed without merging**